components/list/list-item-content.js

```diff
@@ -19,7 +19,7 @@ class ListItemContent extends LitElement {
 			.d2l-list-item-content-text > div {
 				border-radius: var(--d2l-list-item-content-text-border-radius);
 				color: var(--d2l-list-item-content-text-color);
-				display: inline-block;
+				display: inline-grid;
```

Comment:
I found some old StackOverflow posts suggesting that `min-width: 0;` is a for this issue of elements overflowing their containers, but it didn't work for me. The `grid` and `inline-grid` where the only things I found that worked.

Reply:
Applied this to the Work to Do widget and it's truncating it, but still allowing `d2l-list-item-content` to take up 100% width unless I apply `inline-grid` to _it_ as well.

`inline-grid` on `d2l-list-item-content > div`
<img width="328" alt="Screen Shot 2023-02-03 at 3 25 04 PM" src="https://user-images.githubusercontent.com/1289042/216703775-52568b02-1760-44b9-83f2-04d2cd8a3d64.png">

`inline-grid` on `d2l-list-item-content, d2l-list-item-content > div`
<img width="301" alt="Screen Shot 2023-02-03 at 3 25 18 PM" src="https://user-images.githubusercontent.com/1289042/216703810-bee350ed-f361-46f5-9768-535bbbe6a4ba.png">

Reply:
Interesting. Note that it is the inner-most `div` (`.d2l-list-item-content-text > div`) that gets this, not the first `div`. This is what the Quick Eval widget looks like with the fix. I haven't tested Work To Do widget, but I can.
![image](https://user-images.githubusercontent.com/9042472/216705175-b0d1ebf5-5bf2-4b1b-8f7e-902b4e2ab1dc.png)

Reply:
Yeah sorry, typed that out wrong. Edited.

Reply:
Wonder if we are doing something different, or the case I am looking at is slightly different. This is what I get:
![image](https://user-images.githubusercontent.com/9042472/216707071-0777e356-6653-4514-ab0f-f409f7281f6d.png)

Reply:
Ah, it's the secondary text that's keeping it long in my first screenshot above. Applying `inline-flex` to the `d2l-list-item-content-text-secondary` _and_ w2d's own `d2l-w2d-attribute-list` fixes that but that seems not ideal.

Reply:
Ok, I've made a couple more tweaks. I think it should be working for that case now too. I've added a bunch more visual-diff tests to capture this case as well as a couple others.